app/assets/stylesheets/cta.scss

```diff
@@ -15,6 +15,26 @@
   display: flex;
 }
 
+@media (max-width: 40.0525em) {
+  .cta-banner {
+    flex-direction: column;
+  }
+
+  .cta-banner-img {
+    width: 100%;
+    order: 1;
+  }
+
+  .cta-banner-text {
+    order: 2;
+    padding: 20px;
+  }
+
+  .table-scroll {
+    overflow: scroll;
+  }
+}
+
 @media (max-width: 40.0525em) {
   .hf-card {
```

Comment:
👀

Reply:
This can be deleted (it is not being used) 👍  I will include this change in the next commit